lib/cursor.js

```diff
@@ -1,7 +1,36 @@
-import { go, put } from '../vendor/js-csp';
-import { partial } from 'underscore';
+import { isObject, isFunction, partial, defaults, map, filter, find, isArray } from 'underscore';
 import ObjectPath from 'object-path';
+import putOnChan from './put_on_chan';
 
+const hasSameValue = function (valueA, cursorB) {
+  return isEqual(valueA, cursorB.value);
```

Comment:
Comparing a value object to a cursor property seems a bit specific when the helper's name is simply `hasSameValue` ? Should it be `cursorHasSameValue` at least?